front_end/src/components/layout/Navbar.js

```diff
@@ -6,132 +6,219 @@ import { logoutUser } from "../../actions/authActions";
+          <Typography
+            variant="title"
+            color="inherit"
+            className={classes.normalLink}
+          >
+            Abode
+          </Typography>
+          <Button variant="text" color="inherit">
+            Deals of the Week
+          </Button>
+          <Button
+            className={classes.beforeSeperation}
+            variant="text"
+            color="inherit"
           >
-            Logout
-          </a>
-        </li>
-      </ul>
-    );
-
-    const guestLinks = (
-      <ul className="navbar-nav">
-        <li className="nav-item">
-          <Link className="nav-link" to="/register">
-            Sign Up
-          </Link>
-        </li>
-        <li className="nav-item">
-          <Link className="nav-link" to="/login">
+            Popular Cities
+          </Button>
+          <Button
+            className={classes.afterSeperation}
+            variant="text"
+            color="inherit"
+          >
+            Sign up
+          </Button>
+          <Button className={classes.rightMost} variant="text" color="inherit">
             Login
-          </Link>
-        </li>
-      </ul>
-    );
-
-    return (
-      <div>
-        <nav className="navbar navbar-expand-sm navbar-dark bg-dark mb-4">
-          <div className="container">
-            <Link className="navbar-brand" to="/">
-              Home
-            </Link>
-
-            <form onSubmit={this.onSearchClick}>
-              <div className="input-group mr-auto">
-                <input
-                  style={{ height: 36 }}
-                  type="input"
-                  className="form-control"
-                  name="search"
-                  value={this.state.query}
-                  onChange={this.onChange}
-                />
-                <div className="input-group-append">
-                  <Link to="/Search">
-                    <button
-                      className="btn btn-outline-secondary"
-                      type="button"
-                      onClick={this.onSearchClick}
-                      type="submit"
-                    >
-                      Search
-                    </button>
-                  </Link>
-                </div>
-              </div>
-            </form>
-
-            {isAuthenticated ? authLinks : guestLinks}
-          </div>
-        </nav>
-      </div>
-    );
-  }
+          </Button>
+        </Toolbar>
+      </AppBar>
+    </div>
+  );
 }
 
 Navbar.propTypes = {
-  logoutUser: PropTypes.func.isRequired,
-  auth: PropTypes.object.isRequired
+  classes: PropTypes.object.isRequired
 };
 
-const mapStateToProps = state => ({
-  auth: state.auth,
-  query: state.query
-});
-// if this.props.query is empty we will not show the Search page
-
-export default connect(
-  mapStateToProps,
-  { logoutUser, submitQuery, clearCurrentProfile }
-)(Navbar);
+export default withStyles(styles)(Navbar);
+
+// import React, { Component } from "react";
```

Comment:
Is there a reason you left all the old stuff in?

Reply:
I'm not sure if I should delete all of this. I think we should keep it there for now so we can use it as reference once we're connecting the frontend and the backend. I'm afraid that without it, we'd have to re-figure out how to connect redux and backend again.

Reply:
sounds reasonable